library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,38 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for a dependency tree to contain conflicts.
+Unfortunately, as soon as any library in a dependency tree introduces a breaking
+change, consumers can't use the latest versions of those libraries in the
+dependency tree together. To enable clients to depend on the latest versions of
+their dependencies, breaking changes should be propagated as quickly as
+ possible.
+
+Perform the rollout in this manner:
+
+1. Decide whether to break the feature as an atomic change or to perform the
```

Comment:
Decide whether to introduce the incompatibility in a single release or to perform the change over two releases: first deprecating, then deleting.

Reply:
I will rework this in a slightly different way. (Hopefully I address your implicit concerns)